var/spack/repos/builtin/packages/py-pystache/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPystache(PythonPackage):
+    """Pystache is a Python implementation of Mustache. Mustache is a
+    framework-agnostic, logic-free templating system inspired by ctemplate and
+    et."""
+
+    homepage = "https://github.com/sarnold/pystache"
+    git = "https://github.com/sarnold/pystache"
+    pypi = "pystache/pystache-0.6.0.tar.gz"
+
+    maintainers = ["adamjstewart"]
```

Comment:
You don't need to explicitly list me, I'm inherited from the base class

Reply:
Ack, I removed the entry.